users/internal/handlers/users.go

```diff
@@ -86,3 +88,36 @@ func (u Users) GetOne(w http.ResponseWriter, r *http.Request) {
 		log.Error().Err(err).Msg("failed to encode response")
 	}
 }
+
+func (u Users) UpdateUserName(w http.ResponseWriter, r *http.Request) {
```

Comment:
usually method is aimed to update whole user but here it's name only. you better go with general implementation.

Reply:
I planned to create a Put Method where I will update a full user data. So this why I specified "Name" in the function name
https://github.com/MullerJulia/gowork/pull/5